hack/run-ci-e2e-test.sh

```diff
@@ -41,6 +41,42 @@ get_WMCO_logs() {
   done
 }
 
+enable_debug_logging() {
+  if [[ $(oc get -n $WMCO_DEPLOY_NAMESPACE pod -l name=windows-machine-config-operator -ojson) == *"--debugLogging"* ]]; then
+    # debug logging already enabled
+    return 0
+  fi
+  WMCO_SUB=$(oc get sub -n $WMCO_DEPLOY_NAMESPACE --no-headers |awk '{print $1}')
+  oc patch subscription $WMCO_SUB -n $WMCO_DEPLOY_NAMESPACE --type=merge -p '{"spec":{"config":{"env":[{"name":"ARGS","value":"--debugLogging"}]}}}'
+  # delete the deployment to ensure the changes are picked up in a timely matter
+  oc delete deployment -n $WMCO_DEPLOY_NAMESPACE windows-machine-config-operator
+  retries=0
+  debug_logging_enabled=0
+  until [[ $debug_logging_enabled -eq 1 || $retries -gt 30 ]]; do
```

Comment:
/lgtm ! 

My only question is if this is likely to take a whole five minutes? Maybe it would be better to exit sooner if it's taking that long, but I don't think that's a huge deal.

Reply:
It probably won't take 5 minutes, but it's better to wait a bit longer as the penalty for exiting a little too soon is another 1-2 hours for the retest job to get a cluster prepped.